aws-android-sdk-mobile-client/src/main/java/com/amazonaws/mobile/client/AWSMobileClient.java

```diff
@@ -258,8 +258,8 @@
     AWSMobileClientCognitoIdentityProvider provider;
     DeviceOperations mDeviceOperations;
     AmazonCognitoIdentityProvider userpoolLL;
-    private Auth hostedUIJSONConfigured;
-    private Auth hostedUI;
+    Auth hostedUIJSONConfigured;
```

Comment:
were these the changes you were going to revert?

Reply:
Reverted. Sorry it was a miss.